core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -1231,11 +1232,11 @@ class SubmissionSpec(_system: ActorSystem) extends TestKit(_system)
         deleteIntermediateOutputFiles = false
       )
 
-      val ex = intercept[RawlsExceptionWithErrorReport] {
+      val ex = intercept[DataEntityException] {
         Await.result(workspaceService.validateSubmission( minimalTestData.wsName, submissionRq ), Duration.Inf)
       }
-      ex.errorReport.statusCode shouldBe Option(StatusCodes.BadRequest)
-      ex.errorReport.message shouldBe "Reference name unknown does not exist in workspace myNamespace/myWorkspace."
+      ex.code shouldBe StatusCodes.NotFound // TODO: will this code change have any repercussions?
+      ex.getMessage shouldBe "Reference name unknown does not exist in workspace myNamespace/myWorkspace."
```

Comment:
need to follow up on if this behavior change is acceptable

Reply:
is there a corresponding response code in the swagger yaml that would need to change as well?

Reply:
probably! This is where I ran out of time and I didn't track down which APIs would be affected.

Reply:
updated the swagger - this affects validate-submission and validate-configuration.